src/Polysemy/Scoped.hs

```diff
@@ -2,21 +2,86 @@ module Polysemy.Scoped (
   -- * Effect
   Scoped,
 
-  -- * Constructor
+  -- * Constructors
   scoped,
+  scoped_,
+  rescope,
 
   -- * Interpreters
-  runScoped,
-  runScopedAs,
   interpretScopedH,
+  interpretScopedH',
   interpretScoped,
   interpretScopedAs,
+  runScoped,
+  runScopedAs,
 ) where
 
 import Polysemy.Internal (InterpreterFor, Sem, liftSem, raise)
-import Polysemy.Internal.Scoped (Scoped (Run, InScope), scoped, interpretH')
-import Polysemy.Internal.Union (Weaving(Weaving), injWeaving)
-import Polysemy.Internal.Tactics (Tactical, runTactics)
+import Polysemy.Internal.Combinators (interpretH)
+import Polysemy.Internal.Scoped (Scoped (InScope, Run), interpretWeaving, rescope, scoped, scoped_)
+import Polysemy.Internal.Tactics (Tactical, runTactics, runTSimple, liftT)
+import Polysemy.Internal.Union (Weaving (Weaving), injWeaving)
+
+-- |Interpreter for 'Scoped', taking a @resource@ allocation function and a parameterized handler for the plain
+-- @effect@.
+--
+-- >>> runScoped withResource scopedHandler
```

Comment:
Is this code snippet related to `interpretScopedH`?

Reply:
whoopsie!